DigitalLibraryofTennessee/DLTN_XSLT
# Add catalog and modify template for manifest look up.

**GitHub Issue**: [DLTN_XSLT-77](https://github.com/DigitalLibraryofTennessee/DLTN_XSLT/issues/77), [DPLA-209](https://jira.lib.utk.edu/projects/DPLA/issues/DPLA-209)

## What does this Pull Request do?

Adds an initial catalog.xml with just UTC manifested items and modifies the http dc:identifier template to add a manifest link when the document matches an item in catalog.xml

## What's new?

1. A new catalog.xml file that represents all items from UTC qdc with manifests is present. This file is generated with the [ContentDM Catalog Manifest Generator](https://github.com/DigitalLibraryofTennessee/contentdm_catalog_manifest_generator).  The structure of the catalog.xml is designed to make it easy to understand what parts need to be updated or fixed without a full regenerations.
2. Our UTC QDC to MODS template has been modified so that a /mods:location/mods:url[@note="iiif-manifest"] is generated in MODS when the path to the object matches the path to an item in our catalog.xml.  

## How should this be tested?

1. Open Oxygen and test sample objects to see if a manifest is getting added to MODS if it has a matching manifest.
2. Check that it doesn't get one when there isn't a manifest.
3. Check that the manifest link resolves.


## Additional Notes:

Talk to me on Slack in the channel in the DLTN / DPLA channel if needed.

## Interested parties

@mlhale7
